### PR TITLE
Trending word signal sends all words in one signal

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/sanbase_web/endpoint.ex [hunk 1]
```diff
@@ -87,6 +87,12 @@ defmodule SanbaseWeb.Endpoint do
     |> Path.join(word)
   end
 
+  def trending_word_url(words) when is_list(words) do
+    website_url()
+    |> Path.join("labs/trends/explore")
+    |> Path.join(words |> Enum.join(" OR "))
```
Comment (tspenov) [inline lib/sanbase_web/endpoint.ex]: should we `URI.encode` the last part or that will work ?

**Comment by tspenov** [comment]
Can you add a test for the new payload ?

File: lib/sanbase/signals/trigger/settings/trending_words_trigger_settings.ex [hunk 2]
```diff
@@ -183,14 +184,25 @@ defmodule Sanbase.Signal.Trigger.TrendingWordsTriggerSettings do
...
       """
       The word **#{word}** is in the trending words.
 
       More info here: #{SanbaseWeb.Endpoint.trending_word_url(word)}
       """
     end
 
+    defp payload(%{operation: %{trending_word: true}}, [_, _ | _] = words) do
+      {last, previous} = List.pop_at(words, -1)
+      words_str = (Enum.map(previous, &"**#{&1}**") |> Enum.join(",")) <> " and #{last}"
```
Comment (tspenov) [inline lib/sanbase/signals/trigger/settings/trending_words_trigger_settings.ex]: `last` is not bold text

File: lib/sanbase_web/endpoint.ex (hunk 1 above)
Comment (IvanIvanoff) [reply lib/sanbase_web/endpoint.ex]: true, `Path.join` won't alone handle that properly

File: lib/sanbase_web/endpoint.ex [hunk 3]
```diff
@@ -106,6 +114,7 @@ defmodule SanbaseWeb.Endpoint do
     Config.get(:backend_url)
   end
 
+  @spec api_url :: <<_::64, _::_*8>>
```
Comment (tspenov) [inline lib/sanbase_web/endpoint.ex]: lets remove this spec

**Pull request merged**